src/store/modules/notification/index.ts

```diff
@@ -0,0 +1,152 @@
+import { NOTIFICATION_UNKNOWN_ERROR } from '@/setup/globals';
+import i18n from '@/setup/i18n';
+import api from '@/helpers/api';
+import { IMessage, INotification } from '@/types/c-notification';
+import { defineModule } from 'direct-vuex';
+import { moduleActionContext } from '@/store';
+
+interface IModNotificationState {
+  notifications: INotification[];
+  id: number;
+}
+
+/**
+ * Pushes the given notification to the notification stack.
+ *
+ * This is a work around to handle app internal and external pushes of notifications.
+ * A better way would be to refactor this to an action.
+ *
+ */
+function pushNotification(state: IModNotificationState, options: INotification) {
+  if (!options) {
+    return;
+  }
+
+  const notification = {
+    ...options,
+    id: state.id,
+    expire: options.expire !== false,
+    delay: options.delay || 3
+  };
+  const metaData = notification?.message?.meta || { confirmationType: '', id: '' };
+
+  // Attach confirmation actions (if confirmationType is missing, this is ignored)
+  switch (metaData.confirmationType) {
```

Comment:
Please remove the switch. This was client specific.

Reply:
This has been reworked in the develop and already replaced in the remake branch